homeassistant/components/modbus/sensor.py

```diff
@@ -112,6 +113,7 @@ async def async_update(self, now: datetime | None = None) -> None:
         if raw_result is None:
             if self._lazy_errors:
                 self._lazy_errors -= 1
+                async_call_later(self.hass, timedelta(seconds=1), self.async_update)
```

Comment:
Should this be canceled when the entity is removed?

Reply:
Should this be assigned to `self._cancel_call` ?

Reply:
of course, done